History/HistoryModel.h

```diff
@@ -104,6 +114,15 @@ private slots:
     void beforeAppend();
     void afterAppend();
 
+    /**
+     * @internal
+     * @see RM::EventInterface
+     */
+    ///@{
```

Comment:
This will probably uglify the doxygen output a lot - unless used _really_ consistently (and doesn't do what is expected anyway).